modules/ROOT/pages/micrometer-metrics.adoc

```diff
@@ -96,9 +99,8 @@ The following example shows the corresponding `server.xml` file configuration to
   <variable name="mp.metrics.elastic.index" value="micrometer-metrics" />
 
 	<library id="micrometerLibrary">
-		<fileset dir="/path/to/directory/with/micrometer/libraries" includes="*.jar" />
-	</library>
-    
+      <fileset dir="/path/to/directory/with/micrometer/libraries" includes="*.jar" />
```

Comment:
might need to double check the indentation -- lines 101/103 use a tab, line 102 uses spaces

Reply:
Thanks Don, fixed in #6257, [draft site shows the change](https://docs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/docs/latest/micrometer-metrics.html#example). I checked the other docs for indent issues too but didn't find anything. It's hard to see these problems in my VS Code editor! Everything looks ok, but then it looks wacky on the site.